fusor-ember-cli/app/templates/satellite/access-insights.hbs

```diff
@@ -1,6 +1,9 @@
 <div class="row">
   <div class='col-md-6'>
     <p>
+      <a href='https://access.redhat.com/insights/' target='_blank'>Red Hat Insights</a> empowers you to proactively identify configuration risks across your IT environment and resolve them before they impact your business operations. <a href='https://access.redhat.com/insights/' target='_blank'>Click here to learn more.</a>
```

Comment:
I just took description from https://access.redhat.com/insights/
Is there anything else to add?